docs/code/Flakes/Tutorials/Installation.md

```diff
@@ -0,0 +1,40 @@
+---
+title: 01-Installation
+date: 2021-02-09
+---
+
+Presuming nix is already installed, Nix flakes are available from Nix version 2.4 and up.
+
+## Non-NixOS
+
+```
+$ nix-env -iA nixpkgs.nixUnstable # or nixpkgs.nix_2_4
+                                  # or nixpkgs.nix_2_5
+                                  # or nixpkgs.nix_2_6, etc.
+```
+
+Then edit either `~/.config/nix/nix.conf` or `/etc/nix/nix.conf` and add:
+
+```
+experimental-features = nix-command flakes
+```
+
+## NixOS
```

Comment:
NixOS-specific instructions probably aren't necessary. Especially because `nix.extraOptions` has become `nix.settings.experimental-features` in the latest nixos-unstable versions.